src/main/resources/assets/js/models/AppCollection.js

```diff
@@ -14,6 +14,14 @@ var AppCollection = SortableCollection.extend({
     return response.apps;
   },
 
+  filterById: function (searchValue) {
+    var filteredCollection = this.filter(function (app) {
+      return app.id.indexOf(searchValue) > -1;
+    });
+
+    return new AppCollection(filteredCollection);
```

Comment:
This leads to a bug where a filtered collection cannot be sorted anymore in the list component.
Reason is that the sorting information is added to the filtered collection in the list and once the filter is changed the collection gets replaces using the original sorting information again.

A solution to this would be to manage the sorting information directly in the list component and not on the collection. This would also allow different sortings in multiple components.